homeassistant/components/radarr/sensor.py

```diff
@@ -74,12 +74,12 @@ class RadarrSensorEntityDescription(
     """Class to describe a Radarr sensor."""
 
     description_fn: Callable[
-        [RadarrSensorEntityDescription, RootFolder],
-        tuple[RadarrSensorEntityDescription, str] | None,
-    ] = lambda _, __: None
+        [RadarrSensorEntityDescription[T], RootFolder],
+        tuple[RadarrSensorEntityDescription[T], str] | None,
+    ] | None = None
```

Comment:
This is a small logic change. Even an empty lambda would  have evaluated to true.
https://github.com/home-assistant/core/blob/1a9bcafbd209e166e43a66fb3609b3c14eb9da36/homeassistant/components/radarr/sensor.py#L165-L167

@tkdrob If you agree, I can tag this PR for the next beta release.

Reply:
If you want to leave it for next, that's fine. There are tests for these sensors and those checkout. I don't see any harm in putting it in this release or the next.